ruby/src/main/java/com/google/protobuf/jruby/RubyServiceDescriptor.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Protocol Buffers - Google's data interchange format
+ * Copyright 2014 Google Inc.  All rights reserved.
```

Comment:
Nit: 2014 -> 2024.

Reply:
I wasn't actually sure if I was supposed to change those - I've gone and done that now.